develpoedCalculator.js

```diff
@@ -1 +1,8 @@
-class DevelopedCalculator extends Calculator {};
\ No newline at end of file
+class DevelopedCalculator extends Calculator {
+    constructor(selector) {
```

Comment:
you don't need to add `constructor` to this class yet.
`DevelopedCalculator` class is passing the `selector` to its parent automatically